spec/controllers/api/v1/skills_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe Api::V1::SkillsController, type: :controller do
+
+  let(:skill) {create(:skill)}
+
+  describe 'index' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.